### PR TITLE
Add images with all runtimes installed

### PR DESCRIPTION
**Pull request by kili-ilo**

### CODE REVIEW

**Comment by kili-ilo** [comment]
~TODO: Add an image with VS Code + all runtimes (but not all IDEs)~ Done.

File: .github/workflows/on_external_request_to_build_and_publish_all_images.yaml [hunk 1]
```diff
@@ -4,13 +4,13 @@ on: workflow_dispatch
 
 jobs:
 
-  build_and_publish_all_dev_images:
-    uses: itopia-inc/spaces-images/.github/workflows/build_and_publish_all_images.yaml@main
+  build_all_dev:
+    uses: itopia-inc/spaces-images/.github/workflows/build_and_publish_all_images.yaml@add-everything-image
```
Comment (danisla) [inline .github/workflows/on_external_request_to_build_and_publish_all_images.yaml]: This branch reference needs to be reverted back.

File: .github/workflows/on_external_request_to_build_and_publish_all_images.yaml [hunk 2]
```diff
@@ -4,13 +4,13 @@ on: workflow_dispatch
...
     with:
       environment: "dev"
 
-  build_and_publish_all_prod_images:
-    needs: build_and_publish_all_dev_images
-    uses: itopia-inc/spaces-images/.github/workflows/build_and_publish_all_images.yaml@main
+  build_all_prod:
+    needs: build_all_dev
+    uses: itopia-inc/spaces-images/.github/workflows/build_and_publish_all_images.yaml@add-everything-image
```
Comment (danisla) [inline .github/workflows/on_external_request_to_build_and_publish_all_images.yaml]: This branch reference needs to be reverted back.

File: complete-setups/Go-and-VSCode/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Complete Go + GoLand setup
 
-[These Go + GoLand combination images](https://github.com/orgs/itopia-inc/packages?tab=packages&repo_name=spaces-images&q=Go+GoLand)
+[These Go + GoLand combination images](https://github.com/orgs/itopia-inc/packages?tab=packages&repo_name=spaces-images&q=Go+VS+Code)
```
Comment (danisla) [inline complete-setups/Go-and-VSCode/README.md]: The text between the `[ ]` needs to be updated.

**Comment by kili-ilo** [comment]
We now have a second "all-in-one" image with all runtimes but just VS Code (instead of all IDEs)

**Review by danisla — APPROVED** [review]

**Pull request merged**